resources/views/consumables/checkout.blade.php

```diff
@@ -67,16 +81,16 @@
               </div>
             @endif
           <!-- Note -->
-          <div class="form-group {{ $errors->has('note') ? 'error' : '' }}">
+          <div class="form-group {{ $errors->has('checkoutnote') ? 'error' : '' }}">
             <label for="note" class="col-md-3 control-label">{{ trans('admin/hardware/form.notes') }}</label>
             <div class="col-md-7">
-              <textarea class="col-md-6 form-control" id="note" name="note">{{ old('note', $consumable->note) }}</textarea>
-              {!! $errors->first('note', '<span class="alert-msg" aria-hidden="true"><i class="fas fa-times" aria-hidden="true"></i> :message</span>') !!}
+              <textarea class="col-md-6 form-control" id="checkoutnote" name="checkoutnote"></textarea>	      
+              {!! $errors->first('note', '<span class="alert-msg" aria-hidden="true"><i class="fa fa-times" aria-hidden="true"></i> :message</span>') !!}
             </div>
           </div>
         </div> <!-- .box-body -->
         <div class="box-footer">
-          <a class="btn btn-link" href="{{ URL::previous() }}">{{ trans('button.cancel') }}</a>
+          <a class="btn btn-link" href="{{ route('consumables.index') }}">{{ trans('button.cancel') }}</a>
```

Comment:
I updated this since I had a lot of checkouts and wanted to go back to the index by pressing back. but it keeps coming back to previous checkout processes that I did and it doesn't make any sense in my opinion.  maybe I'm missing the idea of the usage of the link, or I was abusing the link during my test.

To revert the implementation on this

Reply:
link reverted back in the last commit